Source/Urho3D/Scene/Node.cpp

```diff
@@ -266,6 +266,24 @@ bool Node::Save(Serializer& dest) const
 }
 
 bool Node::LoadXML(const XMLElement& source)
+{
+    // If we parsing a root file element and it is scene then parse the first node of the scene.
+    XMLElement parent = source.GetParent();
```

Comment:
This thing is... very suspicios. Why this code is even in Node? It should be in prefab. Node doesn't need to know about prefab.
Recursion inside is even more weird.